packages/react/src/components/DataTable/DataTable.js

```diff
@@ -315,9 +315,10 @@ export default class DataTable extends React.Component {
     const checked = rowCount > 0 && selectedRowCount === rowCount;
     const indeterminate =
       rowCount > 0 && selectedRowCount > 0 && selectedRowCount !== rowCount;
-    const translationKey = checked
-      ? translationKeys.unselectAll
-      : translationKeys.selectAll;
+    const translationKey =
+      checked || indeterminate
```

Comment:
```suggestion
      (checked || indeterminate)
```

Does prettier end up taking away the parenthesis here? There seems to be some syntax ambiguity that adding these in might help with. Otherwise, seems great!

Reply:
yeah prettier will remove parentheses there. by default it is evaluated as `(checked || indeterminate) ? ... : ...` rather than `checked || (indeterminate ? ... : ...)`